pkgs/applications/office/beebeep/default.nix

```diff
@@ -0,0 +1,45 @@
+{ lib
+, mkDerivation
+, fetchzip
+, wrapQtAppsHook
+, autoPatchelfHook
+, qtbase
+, qtmultimedia
+, qtx11extras
+}:
+
+mkDerivation rec {
+  pname = "beebeep";
+  version = "5.8.6";
+
+  src = fetchzip {
+    url = "https://netix.dl.sourceforge.net/project/beebeep/Linux/beebeep-${version}-qt5-amd64.tar.gz";
+    sha256 = "sha256-YDgFRXFBM1tjLP99mHYJadgccHJYYPAZ1kqR+FngLKU=";
+  };
+
+  nativeBuildInputs = [
+    wrapQtAppsHook
+    autoPatchelfHook
+  ];
+
+  buildInputs = [
+    qtbase
+    qtmultimedia
+    qtx11extras
+  ];
+
+  installPhase = ''
+    mkdir -p $out/bin
+    install -m755 -D beebeep $out/bin/beebeep
+    autoPatchelf $out/bin/.beebeep-wrapped
```

Comment:
Shouldn't this already be done by autopatchelfHook?

Reply:
If I remember correctly, there were some issues with the with the `.beebeep-wrapped` not being patched.
But it seems that this isn't an issue anymore, so I'm removing it.

Removing `mkdir` and `cp`, however, doesn't build.